### DISCUSSION TITLE
Part Selection

### DISCUSSION THREAD

**Discussion by Tcub** [ozel/DIY_particle_detector#2] [Q&A] [answered]
Question about the part selection. 

Where the through holes chosen due to ease of assembly? I.e. there would be no issue swapping them out for SMD parts would there? (I don't think there would be an issue but I thought it worth just asking)

**Comment by ozel** [comment, marked as answer]
Absolutely! SMD parts, 0805 in particular, work really well on top of the 2.5 mm through-hole footprints. (I've used sometimes 100 nF 0805 if you look closely in the pictures)

The preferred bills of material - as featured with order codes from 4 major suppliers on kitspace.org - focuses on through-hole parts to make it easier for beginners:
https://kitspace.org/boards/github.com/ozel/diy_particle_detector/alpha-spectrometer/
https://kitspace.org/boards/github.com/ozel/diy_particle_detector/electron-detector/

In case of 40 mega Ohm/R3 for the alpha-spectrometer, I've selected an SMD part since it is slightly better noise-wise and through-hole resistors with such high values seem harder to get. 

I'll leave this open for others to ask part related questions.
